packages/extension/src/components/pages/AddNewWallet/ConfirmPassword/ConfirmPassword.tsx

```diff
@@ -47,7 +48,7 @@ export const ConfirmPassword: FC<ConfirmPasswordProps> = ({ setPassword, onConfi
           display: 'flex',
           flexDirection: 'column',
           justifyContent: 'space-between',
-          height: '100%',
+          height: isConnectionFailed ? 'calc(100% - 56px)' : '100%',
```

Comment:
Maybe 56px could be extracted as a const somewhere

Reply:
Good point! I will do this tomorrow!